tests/ssp/main.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       ssp test application
+ *
+ * This test should crash badly when *not* using the ssp module, and panic if
+ * using it.
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+void test_func(void)
+{
+    char buf[16];
+
+    /* deliberately overflowing stack here !! */
+    /* cppcheck-suppress bufferAccessOutOfBounds */
```

Comment:
can you align this with the formatting in https://github.com/RIOT-OS/RIOT/pull/6024 please?